doc/_sphinx/extensions/yarn_lexer.py

```diff
@@ -45,6 +45,7 @@ class YarnLexer(RegexLexer):
         'if',
         'jump',
         'local',
+        'set',
```

Comment:
On the phone it looks like this is tabbed in?

Reply:
Strange, in the code it's regular spaces.